clinica/pipelines/t1_freesurfer_longitudinal/t1_freesurfer_longitudinal_correction_utils.py

```diff
@@ -5,14 +5,28 @@ def init_input_node(caps_dir, participant_id, session_id, long_id, output_dir):
     """Initialize the pipeline."""
     import os
     import errno
+    import datetime
+    import platform
+    from tempfile import mkdtemp
+    from colorama import Fore
     from clinica.utils.longitudinal import read_sessions
     from clinica.utils.ux import print_begin_image
+    from clinica.utils.stream import cprint
 
     # Extract <image_id>
     image_id = '{0}_{1}_{2}'.format(participant_id, session_id, long_id)
```

Comment:
This doesn't concern your changes, but now that we are updated with python >= 3.6, we should start changing all the "string format" by _f-strings_. E.g."
```suggestion
    image_id = f'{participant_id}_{session_id}_{long_id}'.
``